src/polyscope.cpp

```diff
@@ -156,6 +156,10 @@ void pushContext(std::function<void()> callbackFunction, bool drawDefaultUI) {
   // Make sure the window is visible
   render::engine->showWindow();
 
+  if (options::giveFocusOnShow) {
```

Comment:
Could we move this to around L729 (just before the `pushContext()` in `show()`)?

Probably won't matter, but there are some other places `pushContext()` gets called in addition to inside `show()`, so putting the `focusWindow()` call actually inside `show()` will be more precise.